PF-Assignment1/1.BinaryToOctal/Assignment1.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Assignment1
+ *
+ * Date: 13/07/2017
+ *
+ */
+
+public class Assignment1 {
+	
+	/*
+	 *  Converts Binary Number to Octal
+	 *  @param n int
+	 *  @return int octal of binary number
+	 */
+	int convertBinaryToOctal(int n){
+		int x,y,z;	
+		//temporary variables to hold modulus and quotient
+		
+		int groupThree=1; 
+		int iterate;
+		int octal=0;
+		
+		x=n;
```

Comment:
No need to copy the number into temporary variables as it is not used further anywhere.